app/scripts/athena_partition_archive.py

```diff
@@ -520,22 +617,37 @@ def main():
                                 LOGGER.info('Ignoring %s.%s until the 1st of the month.', database_name, table_name)
                             else:
                                 retention = str(TWOMONTHSPLUSCURRENT)
-                                partition(database_name, table_name, s3_location, retention, drop_only)
+                                #partition(database_name, table_name, s3_location, retention, drop_only)
+                                for parts in get_partitions(database_name, table_name, retention):
+                                    create_partition(parts, database_name, f'{table_name}_archive')
+                                    for desc in parts:
+                                        del desc['StorageDescriptor']
+                                    execute_glue_api_delete(database_name, table_name, parts)
                         elif retention_period == '30Days':
                             retention = str(THIRTYDAYS)
-                            partition(database_name, table_name, s3_location, retention, drop_only)
+                            #partition(database_name, table_name, s3_location, retention, drop_only)
+                            for parts in get_partitions(database_name, table_name, retention):
+                                create_partition(parts, database_name, f'{table_name}_archive')
+                                for desc in parts:
+                                    del desc['StorageDescriptor']
+                                execute_glue_api_delete(database_name, table_name, parts)
                         elif retention_period == '30DaysDropOnly':
                             retention = str(THIRTYDAYS)
-                            drop_only = True
-                            partition(database_name, table_name, s3_location, retention, drop_only)
+                            #drop_only = True
+                            #partition(database_name, table_name, s3_location, retention, drop_only)
+                            for parts in get_partitions(database_name, table_name, retention):
+                                for desc in parts:
+                                    del desc['StorageDescriptor']
+                                execute_glue_api_delete(database_name, table_name, parts)
                         elif retention_period == 'PartitionMaxDate':
                             days_to_keep = row["days_to_keep"]
                             retention = (datetime.date.today() - datetime.timedelta(days=int(days_to_keep)))
                             partitioned_by = row["partitioned_by"]
-                            partition_max_date(database_name, table_name, s3_location, retention, partitioned_by)
+                            partition_max_date(database_name, table_name,
+                                               s3_location, retention, partitioned_by)
 
 
-        LOGGER.info("Were done here.")
+        LOGGER.info("We are done here.")
```

Comment:
I love the fact you fixed this typo.
I hope it doesn't mess up people's searches.